tests/test_utils.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+"""Tests for coronawhy_vt utilities."""
+
+import unittest
+from math import inf
+
+# Project specific packages
+from coronawhy_vt.utils.local_processing import make_ngram_map_and_list,\
+    pull_mentions_direct,\
+    pull_mentions_lemmatized,\
+    run_replace,\
+    same_sentence_check,\
+    title_blocks
+from tests.constants import PDF_JSON_DOCUMENT_PARSES, SAMPLE_SENTENCE, STUDY_WORDS
+from tests.utils.utils import get_text, list_compare
+
+
+class TestUtils(unittest.TestCase):
+    """Main class for testing utilities."""
+
+    # TODO Update Travis CI file to include this once tests are stable.
+    # TODO add logging debug args
+
+    @classmethod
+    def setUp(cls):
+        """Set up test class."""
+        # TODO initialize class when we refactor the functions into classes
+        cls.expected_virus_words = ['COVID-19', 'COVID 19', '2019-nCoV', '2019 nCoV',
+                                    'SARS-CoV-2', 'SARS-CoV 2', 'SARS CoV-2', 'SARS CoV 2',
+                                    'SARS-CoV2', 'SARS-CoV19', 'SARS-CoV2019', 'SARSCoV2',
+                                    'SARSCoV 2', 'SARSCoV19', 'SARSCoV2019', 'SARS CoV2',
+                                    'SARS Cov19', 'SARS Cov 19', 'SARS CoV 2019', 'SARS CoV2019',
+                                    '2019 Coronavirus', 'Coronavirus Disease 2019',
+                                    'coronavirus 2019', 'novel coronaviruses', 'covid19']
+
+    def test_make_ngram_map_and_list_study_words(self):
+        """Test make_ngram_map_and_list on study words."""
+        study_names, study_ngram_map = make_ngram_map_and_list(STUDY_WORDS, min_word_len=0, max_num_tokens=inf)
+
+        expected_study_names = ['vitro', 'in-vitro', 'cell_culture', 'fbs', 'fetal_bovine_serum',
```

Comment:
better style to have a newline after the open bracket `[` and at the end before the close bracket `]`